### ISSUE TITLE
Pester.bat reference in Test Results

### ISSUE THREAD

**Issue by fflaten** [pester/docs#200]
### Checklist

- [X] Issue has a meaningful title
- [X] I have searched the existing issues. See [issues](https://github.com/pester/docs/issues)

### Link to page

https://pester.dev/docs/usage/test-results#using-binpesterbat

### What is the issue?

v5 Test Results docs reference bat-file. Something in TeamCity specifically or the v3/v4 pester.bat that was should remove?

**Comment by bravo-kernel** [comment]
@nohwnd please advise

**Comment by nohwnd** [comment]
Yes you are right, Pester.bat is gone, and I don't think we saw any issues around TeamCity or AppVeyor in the past few years. Maybe both of those sections can be removed?

**Comment by fflaten** [comment]
👍 Should probably change all CI-examples to use advanced configuration since legacy set is deprecated.

Maybe also include a sample for using the result-object for custom console output. If not done already, I might look at it when NUnit3 is ready.

**Comment by nohwnd** [comment]
I good first issued it, and tweeted it out.
